presto-docs/src/main/sphinx/release/release-0.148.rst

```diff
@@ -21,8 +21,10 @@ General Changes
   honored by node scheduler. This bug could stop the cluster from making further progress.
 * Fix handling of ``INTERVAL DAY TO SECOND`` type in REST API. Previously, intervals greater than
   ``2,147,483,647`` milliseconds (about ``24`` days) were returned as the wrong value.
+  Additionally, negative intervals were formatted incorrectly and could not be read.
 * Fix handling of ``INTERVAL YEAR TO MONTH`` type in REST API. Previously, intervals greater than
   ``2,147,483,647`` months were returned as the wrong value.
+  Additionally, negative intervals were formatted incorrectly and could not be read.
```

Comment:
No, this is another sentence as part of the same list item, so I don't think repeating the full type name is necessary.

Or should I split them out into a single, separate item?